ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -412,6 +412,30 @@ class SignalSpec: QuickSpec {
 				expect(lastValue).to(equal("2"))
 			}
 		}
+		
+		
+		describe("mapError") {
+			it("should transform the errors of the signal") {
+				let (signal, sink) = Signal<Int, TestError>.pipe()
+				let producerError = NSError(domain: "com.reactivecocoa.errordomain", code: 100, userInfo: nil)
+				let mappedSignal = signal |> mapError { _ in
+					producerError
+				}
+
+				var error: NSError?
+
+				mappedSignal.observe(error: {
```

Comment:
You can avoid `mappedSignal` and pipe observer as well `signal |> mapError {...} |> observer {...}`